test/componentprocessor/componentprocessor.test.js

```diff
@@ -98,7 +98,7 @@ describe.only("component parsing support", () => {
 
                 const expectedPrefix = "pkg:/test/componentprocessor/resources/scripts/";
                 let expectedScripts = ["extendedComponent.brs", "utility.brs"].map(scriptName =>
-                    path.join(expectedPrefix, scriptName)
+                    path.join(expectedPrefix, scriptName).replace(/\\/g, "/")
```

Comment:
This test was failing on my Windows computer. It's unrelated to this PR, but was a one-liner, figured I would include it. I can pull it out into its own PR if you prefer.

Reply:
Strange!  A separate PR would be lovely — I suspect there's a few cases where we use `path.join()` in an attempt to be portable when we actually want `path.posix.join()` https://nodejs.org/api/path.html#path_windows_vs_posix , and it'll be easier to review those in isolation 😄

Reply:
k. I'll submit a separate PR with just this fix in it.